services/navigation/builtin/builtin.go

```diff
@@ -373,7 +389,15 @@ func (svc *builtIn) startWaypoint(ctx context.Context, extra map[string]interfac
 		defer svc.activeBackgroundWorkers.Done()
 
 		navOnce := func(ctx context.Context, wp navigation.Waypoint) error {
-			_, err := svc.motion.MoveOnGlobe(
+			svc.logger.Debugf("MoveOnGlobe called going to waypoint %+v, heading: %f, movementSensor: %s, obstacles: %v, motionCfg: %#v, extra: %s",
```

Comment:
Logging nav's calls to MoveOnGlobe.

Reply:
I think this makes more sense in motion's logs

Reply:
moved what I could. The waypoint still needs to be logged here so we know the waypoint id (motion doesn't get a waypoint, it gets a geopoint).